src/LiveComponent/tests/Fixtures/Form/FormWithManyDifferentFieldsType.php

```diff
@@ -74,6 +74,28 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'foo' => 1,
                 ],
             ])
+            ->add('choice_required_with_preferred_choices_array', ChoiceType::class, [
```

Comment:
Maybe an empty array of preferred choices just to be triple sure ? 

(i know this is the case without setting it -- aka the other tests, but this will prevent any regression if ChoiceType internals / view data change)